.nvmrc

```diff
@@ -1 +1 @@
-lts/erbium
+lts/fermium
```

Comment:
Do we want to upgrade to Node 14 as part for this?

Reply:
No Ben, I thought to fix the build issue by making changes here. Since it is not working I am reverting this back to erbium. You can see changes [here](https://github.com/cerner/terra-ui/pull/429/commits/6a6bc6a3d076cfd5eef5149bf92a8b179caff6b9).